index.md

```diff
@@ -0,0 +1,24 @@
+Uganda's Emergency Services Technical Guide 
```

Comment:
@AustinBwanika Delete this file not needed

Reply:
@ssmusoke Okay I will delete the File